### ISSUE TITLE
Runtime Error in 30224, 5 - several times right before/after game-end in Fritz GUI.

### ISSUE THREAD

**Issue by oherbst** [tthsqe12/asm#25]
Recently had several exceptions with current pedant- and asmFish (2016-09-25), mostly on game-ends / draws. Not sure if it is really an exception in the engine, or maybe just Fritz gui:

![runtime error in 30224 5](https://cloud.githubusercontent.com/assets/22038957/18901807/5ff48cd6-854e-11e6-9ce6-2320eca10e92.png)

Once or twice an exception also occured in the mid of the game and the whole Fritz GUI crashed. 
Never had this problem with build from 2016/08/31.

**Comment by oherbst** [comment]
Windows Error-Log:
Name of defective Program/Module:
 Name des fehlerhaften Moduls: ChessProgram13.exe, Version: 13.39.0.0, Zeitstempel: 0x54ed82d8

Exception-Code:
Ausnahmecode: 0xc0000005

Error-Code / Offset:
Fehleroffset: 0x0049f754

ID of defective process:
ID des fehlerhaften Prozesses: 0x16ac

Fritz 13 GUI is most current version. Google search for this error yielded no result.

**Comment by oherbst** [comment]
Still some more runtime-errors (with Low-Process Priority for Fritz-GUI and without Large Pages). Same issues, it crashes at the end of the game, sometimes in the middle:

![runtime error in 30258 270](https://cloud.githubusercontent.com/assets/22038957/18918473/3b781754-859b-11e6-8673-af4e9e0b6beb.png)

As I see no indication for any other problems and the problems appeared since build 2016-09-25, I believe it is caused by the asm-engine.

**Comment by Rechenschieber** [comment]
Your error log says the GUI crashed, not the engine.

**Comment by oherbst** [comment]
Yes, it was the GUI crashing on Windows 10. Apparently there was some problem with the MS Visual C++ runtime redistributable on Windows 10 Build 1511. I had discovered some other programs crashing as well.

After some problems with the 1607 Upgrade via Windows Update I meanwhile managed to successfully install Windows 10 Build 1607 plus latest updates.
